guide/how-it-works/liquidity.md

```diff
@@ -306,6 +306,48 @@ Consider how you can help the user with channel management without them even kno
 
+## Channel reserve
+
+### What is a channel reserve?
+
+A channel reserve is an amount that is set aside by each channel participant which ensures neither have 'nothing at stake' if a cheating attempt occurs. This reserve can not be spent, and is held aside for the entirety of the channels lifetime.
+
+Channel reserves make cheating attempts less economical. When one channel party attempts to cheat the other and they are caught, a [penalty transaction](https://fiatjaf.com/73095980.html) can be used to steal all the cheating parties bitcoin as punishment. Channel reserves makes it so cheating attempts always have something at stake making this less likely to occur.
+
+### What is the reserve amount?
+
+The channel reserve amount is dynamic and unique to each channel participant. As defined in [BOLT 2](https://github.com/lightning/bolts/blob/master/02-peer-protocol.md), the channel reserve amount dynamically trends towards 1% of the users local channel capacity. The channel reserve can not be lower than the current 354 sats minimum.
+
+So if a user has 100,000 sats of local capacity, their channel reserve will be 1000 sats (1% of 100,000). This means the user can only spend 99,000 sats of the local capacity. The channel counter party also has their own channel reserve which aims for a 1% reserve. This means the total channels capacity will have around 2% put aside and undependable as a reserve.
+
+As users send and receive funds the channel reserve will dynamically adjust so its always close to 1%. If a users spends 10,000 sats of their 100,000 sats local capacity, their new channel reserve will dynamically adjust from 1000 sats (1% of 100,000 sats) to 900 sats (1% of 90,000). This adjust upwards if the user receive funds.
+
+### How does this effect users?
+
+If a user funds a channel and they are unable to spend the full amount they have put into the channel this can be confusing to them.
+
+From a UX standpoint we would need to consider the following:
+
+<td>Adding funds</td>
+We should ensure that the user is educated so that:
+
+1) When they are adding funds they know that a small amount will be held as a reserve.
+2) They understand that although they cannot spend 100% of their balance they still own all of their funds.
+
+### Additional Considerations
+
+#### Multiple channels
+Each additional channel has its own channel reserve. For example, a user with 20 channels may have more funds locked in reserve than a user with only one channel.
+#### Onboarding
+During onboarding when a user opens their first channel and when trying to send all their funds.
+In general the channel reserve itself adds an additional layer of complexity to mobile users who just want to make payments.
+
+There are however other, much more fool-proof ways to prevent theft such as:
+
+1. [Using watchtowers](https://bitcoin.design/guide/how-it-works/lightning-services/)
```

Comment:
Both bullet lists on this page could be just sentences. They contain just two short items each, not sure much is gained by formatting them as lists.

Reply:
Great. Have removed this formatting to sentences.